distribution/test/performance/jmeter/catalog-transformer-xml/pom.xml

```diff
@@ -0,0 +1,60 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+/**
+ * Copyright (c) Codice Foundation
+ *
+ * This is free software: you can redistribute it and/or modify it under the terms of the GNU Lesser General Public License as published by the Free Software Foundation, either
+ * version 3 of the License, or any later version.
+ *
+ * This program is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+ * See the GNU Lesser General Public License for more details. A copy of the GNU Lesser General Public License is distributed along with this program and can be found at
+ * <http://www.gnu.org/licenses/lgpl.html>.
+ *
+ **/
+-->
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+    <parent>
+        <groupId>ddf.test.performance.jmeter</groupId>
+        <artifactId>jmeter</artifactId>
+        <version>2.9.0-SNAPSHOT</version>
+    </parent>
+    <artifactId>ddf-test-performance-jmeter-catalog-transformer-xml</artifactId>
+    <name>DDF :: Test :: Performance :: JMeter :: Catalog XML Transformer</name>
+    <packaging>pom</packaging>
+    <profiles>
+        <profile>
+            <id>jmeter</id>
+            <activation>
+                <activeByDefault>false</activeByDefault>
+            </activation>
+            <build>
+                <plugins>
+                    <plugin>
+                        <groupId>com.lazerycode.jmeter</groupId>
+                        <artifactId>jmeter-maven-plugin</artifactId>
+                        <version>1.10.1</version>
+                        <executions>
+                            <execution>
+                                <id>jmeter-tests</id>
+                                <phase>verify</phase>
+                                <goals>
+                                    <goal>jmeter</goal>
+                                </goals>
+                                <configuration>
+                                    <propertiesUser>
+                                        <ddf.host>${ddf.host}</ddf.host>
```

Comment:
@pklinef Easy enough, but just to be clear, do you mean something like:
a. `<ddf.host>${target.host}</ddf.host>` (so only the properties we pass through mvn gets changed) or
b. `<target.host>${target.host}</target.host>` (so I should also change the internal JMeter properties)
Also, I would assume that we'd do the same for the dib equivalents?
Also